jameel-institute/daedalus.api
# Jidea-113 real interventions

This branch sets `interventions` in the results using the real closure info from the package results. We return interventions as an array, although currently there will only be one closure period specified in the results. If the response strategy specified was 'none' we return an empty array, since the closure info provided in the result assumes that no closure in fact occurs. 

To test:
1. `./docker/build`
2. `./docker/run_containers`
3. Run model.
Test for elimination response:
```
curl -H "Content-Type: application/json" -d '{"modelVersion": "0.0.1", "parameters": {"country": "Canada", "pathogen": "sars_cov_1", "response": "elimination", "vaccine": "low", "hospital_capacity": "10500"}}' http://localhost:8001/scenario/run
```

Test for "none" response (should be no interventions returned):
```
curl -H "Content-Type: application/json" -d '{"modelVersion": "0.0.1", "parameters": {"country": "Canada", "pathogen": "sars_cov_1", "response": "none", "vaccine": "low", "hospital_capacity": "10500"}}' http://localhost:8001/scenario/run
```
4. ` curl http://localhost:8001/scenario/results/[RUN_ID]` with run Id returned by 3.
